src/test/java/racingcar/model/CarTest.java

```diff
@@ -0,0 +1,38 @@
+package racingcar.model;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import racingcar.util.Direction;
+
+class CarTest {
+
+    private final Car car = new Car();
+    private final Car testCar = new Car("testCar");
+
+    @Test
+    @DisplayName("자동차 이름이 5글자 이하인 경우 이름 리스트 반환")
+    void 이름_수_제한_통과_테스트() {
+        Assertions.assertThat(car.splitByDelimiter("four,fivee"))
+                .hasSize(2)
+                .extracting(Car::getName)
+                .contains("four", "fivee");
+    }
+
+    @Test
+    @DisplayName("자동차 이름이 5글자 초과인 경우 IllegalException 반환")
+    void 이름_수_제한_초과_테스트() {
+        Assertions.assertThatThrownBy(() -> {
+            car.splitByDelimiter("forky,eundeang");
+        }).isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @Test
+    @DisplayName("자동차는 전진 상태일때 자신의 이동 현황에 -를 추가한다.")
+    void 전진_상태_일때_저장값_검증_테스트(){
+        testCar.moveToOrientedDirection(Direction.FORWARD);
+        Assertions.assertThat(testCar.getMoveStatus())
+                .isEqualTo("-");
+    }
+
+}
```

Comment:
RacingManager에 대한 테스트도 진행해보시면 좋을 것 같습니다. 
```suggestion
    @Test
    public void testFindFastestCar() {
        // 임의값 넣어주기
        Car car1 = new CarStub("pobi", "----");
        Car car2 = new CarStub("crong", "------");
        Car car3 = new CarStub("jjubi", "----");

        List<Car> cars = Arrays.asList(car1, car2, car3);

        // 메서드 실행하기
        List<Car> winners = racingManager.findFastestCar(cars);

        // 예상값대로 나오는지 확인하기
        assertEquals(1, winners.size());
        assertTrue(winners.contains(car2));
    }
}
```
이런 식으로 단위 테스트 진행해보시면 좋을 것 같습니다.

Reply:
넵! 테스트 코드 관련해서는 조금 더 깊게 파봐야겠어요!! 좋은 의견 감사합니다!